### ISSUE TITLE
Tagging system

### ISSUE THREAD

**Issue by cosmicBboy** [the-sumit/sdsn-forum#62]
Use this Bootstrap guide to implement a system for tags and for inviting Users on an input element

**Comment by nshimiye** [comment]
This meteor package is too old to be trusted, I'll look for other better options. I am probably going to use the actually bootstrap-typehead library instead

**Comment by JustWalters** [comment]
Can users add new tags for posts, or are they all predefined?

**Comment by cosmicBboy** [comment]
yes, they can add new tags (basically inserts to the Tags collection)

**Comment by cosmicBboy** [comment]
needs styling and sorting option
